gradle/scripts/publish-snapshot.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'maven-publish'
+
+def repositoryDir = file("${rootProject.projectDir}/build/repos/maven")
+
+tasks.register('publishLocalAll') {
+    ext.isLocalPublish = true
```

Comment:
Since environment variable is defined here, do we still need to hardcode `isLocalPublish` in `gradle.properties`?

Reply:
OK